velox/functions/sparksql/RegisterCompare.cpp

```diff
@@ -40,6 +50,10 @@ void registerCompareFunctions(const std::string& prefix) {
       {prefix + "between"});
   registerFunction<BetweenFunction, bool, float, float, float>(
       {prefix + "between"});
+  registerFunction<BetweenFunction, bool, int64_t, int64_t, int64_t>(
+      {prefix + "between"});
+  registerFunction<BetweenFunction, bool, int128_t, int128_t, int128_t>(
```

Comment:
Update the document `velox/docs/functions/spark/comparison.rst`

Reply:
These functions are already added in doc, even previously uses Presto implementation. Refer [comparison.rst](https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/spark/comparison.rst).

Reply:
Please update the supported type https://github.com/facebookincubator/velox/blob/main/velox/docs/functions/spark/comparison.rst?plain=1#L9

Reply:
updated.